extensions/amp-autocomplete/0.1/amp-autocomplete.js

```diff
@@ -158,10 +165,21 @@ export class AmpAutocomplete extends AMP.BaseElement {
     this.templates_ = Services.templatesFor(this.win);
 
     /**
-     * The reference to the <template> tag provided as a child.
-     * @private {?Element}
+     * Whether a <template> or <script type="text/plain"> tag is present.
+     * @private {boolean}
+     */
+    this.hasTemplate_ = false;
+
+    /**
+     * @private {?../../../src/ssr-template-helper.SsrTemplateHelper}
+     */
+    this.ssrTemplateHelper_ = null;
+
+    /**
+     * Whether server-side rendering is supported.
+     * @private {boolean}
```

Comment:
This is very soon going to change from a `boolean` to a `Promise<boolean>`. It may require a decent amount of restructuring, but can you make that change now before submitting? (wrap the call to `.isSupported()` in a `Promise.resolve()`).